app/views/publishers/vacancies/build/job_location.html.slim

```diff
@@ -15,7 +15,10 @@
 
         = f.govuk_collection_radio_buttons :job_location, t("helpers.options.publishers_job_listing_job_location_form.job_location.#{current_organisation.group_type}"), :first, :last
 
-        = f.govuk_submit t("buttons.continue")
+        .govuk-button-group
```

Comment:
This is a little bit of duplication on two step views for now (this one and `job_role`, with the plan being to refactor once we know whether or not all submit buttons will just say "Continue" from now on)